ORM/File.cs

```
@@ -12,21 +13,98 @@ class File
 
         private string path;
         private List<Person> people = new List<Person>();
-        private Person person;
-
+  
         public string Path => path;
-        public Person Person => person;
+
+        public int Count => people.Count;
 
         public List<Person> People => people;
 
         public File(string path, Person person)
         {
             this.path = path;
-            this.person = person;
+            people.Add(person);
+  
+        }
 
+        /// <summary>
+        /// add new person for listpeople
+        /// </summary>
+        /// <param name="person"></param>
+        public void Create(Person person)
+        {
+            int index = people.FindIndex(p => p.Id == person.Id);
+            if(index == -1)
+                people.Add(person);
+            else
+                Console.WriteLine("user with this id already registered");
+        }
+
+        /// <summary>
+        /// Refresh person data in file
+        /// </summary>
+        /// <param name="person">objc link Person</param>
+        public void Update(Person person)
+        {
 
+            int index = people.FindIndex(p => p.Id == person.Id);
+            if (index != -1)
+                people[index] = person;
+          
         }
 
+        /// <summary>
+        /// Remove person from list
+        /// </summary>
+        /// <param name="person">object link Person</param>
+        public void Delete(Person person)
+        {
+            people.Remove(person);
+        }
+
+        /// <summary>
+        /// Get person for id from list 
+        /// </summary>
+        /// <param name="id">id Person</param>
+        /// <returns>if checkPerson = true - return object from list/ otherwise - return empty object</returns>
```

i dont like returning empty object. Lets throw an exception or return null in this case.